### ISSUE TITLE
<score> issues

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#260]
Taking a look at the \<score\> extension I have noticed that the html generated files do not seem to work as expected

The first point is that even with the big bundle download (incorporating ImageMagik and others), the lilypond and timidity software is not included - is this a licencing issue?

Once I installed these two packages (Windows version). The  Gui was able to generate the scores correctly.
However, these files are placed in a subdirectory `xowa\file\en.wikipedia.org\lilypond`

The current images database(s) only know how to deal with `orig` and `thumb`

I also notice that .midi files are produced - I believe at the moment sound files are not incorporated into the downloads - is this a space issue?

**Comment by gnosygnu** [comment]
> Taking a look at the <score> extension I have noticed that the html generated files do not seem to work as expected

Yeah, these seemed like low-frequency items, so I never added support for them. The other complexity was generating these lilypond images as they are slow to build and I didn't want to affect the main dump. So, I was planning on adding a separate command to queue them and run them, but never got round to it.

I checked now, and it looks like they're being used more regularly now: https://en.wikipedia.org/wiki/Special:PagesWithProp/score?propname=score&sortbyvalue=1

Let me reinvestigate and see what's needed. I actually liked this feature at the time, and was sort of disappointed that it was so little used.

> The first point is that even with the big bundle download (incorporating ImageMagik and others), the lilypond and timidity software is not included - is this a licencing issue?

Nope. Just a space vs usage issue. The lilypond / timidity runtime would have added 25 or so MB to the XOWA download package, and it didn't seem it was used anywhere. As per above, I'll reinvestigate.

> However, these files are placed in a subdirectory xowa\file\en.wikipedia.org\lilypond

Yeah, that's by design. XOWA saves a lot of the file artifacts in /file/wiki_name/file_type . Keep in mind these files need to be saved somewhere in the file system for the HTML browser to show them in the &lt;img tag (I'm not a fan of data-uris; for example, src="data:image/png"...)

> I also notice that .midi files are produced - I believe at the moment sound files are not incorporated into the downloads - is this a space issue?

More usage and code-necessity issue, like the lilypond images above. Let me see what's needed to generate these as well.
